docs/finality-provider-operation.md

```diff
@@ -589,29 +584,30 @@ this feature is implemented.
 
 ### 5.3. Jailing and Unjailing
 
-A finality provider can be jailed for the following reasons:
-1. Missing Votes
-   - Not submitting finality votes for a certain number of blocks
-   - Missing votes when the FP has positive voting power
+A finality provider can be jailed for missing votes when they have voting power. 
+However, it's important to understand that:
 
-2. Missing Public Randomness
-   - Not committing public randomness for blocks
-   - Required before voting can occur
+1. A finality provider must commit public randomness for a block before they can 
+have voting power for that block
+2. If a finality provider fails to commit public randomness for a block, they 
+will have zero voting power for that block
+3. Having zero voting power means they cannot vote (and thus cannot miss votes) 
+for that block
+4. Therefore, missing public randomness commitments indirectly prevents voting 
+but does not directly cause jailing
 
-The specific parameters specifying the exact metrics that are taken
-into account for jailing and the period of unjailing
-is controlled by the Babylon chain governance.
+The specific metrics for jailing (e.g., number of missed votes) are defined by 
+the finality module [parameters](https://github.com/babylonlabs-io/babylon/blob/main/proto/babylon/finality/v1/params.proto).
```

Comment:
The details of how the jailing mechanism works can be found [here](https://github.com/babylonlabs-io/pm/blob/b25a0d9f3685bd8e71256926eac63c4f4c678efa/adr/adr-025-remove-voting-power-from-jailed-fps.md). However this is a private repo and we should adapt it in the fp repo or the babylon repo. This is also a TODO for me.